genesyscloud/recording_media_retention_policy/resource_genesyscloud_recording_media_retention_policy_schema.go

```diff
@@ -144,6 +144,12 @@ func ResourceMediaRetentionPolicy() *schema.Resource {
 				Default:     nil,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"team_ids": {
+				Description: "Teams to match conversations against",
+				Type:        schema.TypeList,
```

Comment:
TypeSet is probably better to use here. We've had to change a few fields from TypeList to TypeSet recently because if the API returns the list in a different order, we'll see a plan not empty error. This doesn't happen with TypeSet though because it doesn't care about the ordering of items.